website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -32,6 +32,28 @@ function HelloCodeTitle(props) {
 
 </BrowserWindow>
 
+## Code line numbering
+
+You can enable line numbering for your code block by using `showLineNumbers` key within the language meta string (don't forget to add space directly before the key).
+
+````md
+```jsx showLineNumbers
+function PageLayout(props) {
+  return <Layout title="Awesome Docusaurus page" description="Just one more awesome page on my Docusaurus site">;
```

Comment:
this example is not ideal, maybe add line breaks + find a way to use line highlighting + line numbers in doc?

Reply:
I deliberately made this example with long line of code to show how line numbering will be positioned when scrolling.  About adding line highlighting alongside `showLineNumbers`, it's good idea.

Reply:
I see thanks

What about making the doc example simpler (not scrolling) and adding all edge cases to a dogfood doc to make review easier? (+10 lines, highlighting, scrolling...)